### PR TITLE
Zen mode but only on the client

### PR DESCRIPTION
**Pull request by vEnhance**
If this causes a deadlock then, uh, I have no idea, since it only touches the client.

### CODE REVIEW

**Comment by Zamiell** [comment]
can you keep the npx?
there's no yarn equivalent of npx so its idiomatic in typescript projects to use it even if you are not using npm

File: packages/client/src/globals.ts [hunk 1]
```diff
@@ -58,6 +58,8 @@ export class Globals {
   ui: HanabiUI | null = null;
   // Used to keep track of how many in-game chat messages are currently unread.
   chatUnread = 0;
+  // Zen mode
```
Comment (Zamiell) [inline packages/client/src/globals.ts:61]: get rid of the comment since it doesn't explain anything

Comment (Zamiell) [reply packages/client/src/globals.ts:61]: or alternatively make it explain what zen mode is

**Comment by vEnhance** [comment]
> can you keep the npx? there's no yarn equivalent of npx so its idiomatic in typescript projects to use it even if you are not using npm

Ah, okay. should be done.

**Comment by Zamiell** [comment]
awesome, thanks evan as always

**Pull request merged**